lib/mixlib/install/backend/bintray.rb

```diff
@@ -152,14 +178,16 @@ def parse_platform_info(artifact_map)
           filename = artifact_map["name"]
           architecture = if %w{ x86_64 amd64 x64 }.fuzzy_include?(filename)
                            "x86_64"
-                         elsif %w{ i386 x86 }.fuzzy_include?(filename)
+                         elsif %w{ i386 x86 i86pc i686 }.fuzzy_include?(filename)
                            "i386"
-                         elsif %w{ powerpc }.fuzzy_include?(filename)
+                         elsif %w{ powerpc sun4u solaris2.5.10 solaris2.5.9 }.fuzzy_include?(filename)
```

Comment:
There is no such thing as Solaris with `powerpc` arch so this should remain:

```ruby
elsif %w{ powerpc }.fuzzy_include?(filename)
```
further down you need to make the sparc section:

```ruby
elsif %w{ sparc sun4u sun4v solaris2.5.9 }.fuzzy_include?(filename)
```

`solaris2.5.10` is actually `i386` so it should be in that final section:

```ruby
elsif %w{ msi solaris2.5.10 }.fuzzy_include?(filename)
```

In the future you can use https://github.com/sethvargo/chef-sugar/blob/master/lib/chef/sugar/architecture.rb as a guide along with the `omnibus.*` properties attached to published artifacts in artifactory.chef.co.